runtime/src/main/java/org/corfudb/runtime/view/stream/AbstractQueuedStreamView.java

```diff
@@ -202,7 +202,16 @@ protected ILogData getNextEntry(QueuedStreamContext context,
      *
      * @param address       The address to read.
      */
-    protected abstract @Nonnull ILogData read(final long address);
+    protected @Nonnull ILogData read(final long address) {
+        try {
+            return readImpl(address);
```

Comment:
Let's not create xxImpl methods, please just catch the exception in the BackpointerStreamView. Other implementations (if we ever have any) may have very different ways of recovering.